src/libs/PolicyUtils.ts

```diff
@@ -520,10 +522,33 @@ function getDefaultApprover(policy: OnyxEntry<Policy>): string {
 /**
  * Returns the accountID to whom the given employeeAccountID submits reports to in the given Policy.
  */
-function getSubmitToAccountID(policy: OnyxEntry<Policy>, employeeAccountID: number): number {
+function getSubmitToAccountID(policy: OnyxEntry<Policy>, expenseReport: OnyxEntry<Report>): number {
```

Comment:
The only possibly annoying thing with this change is that now, we're getting the `submitTo` from the report owner ALWAYS - i believe we USED TO be able to pass any employee accountID & get their `submitTo` 🤔 

Technically that may not be a problem if this works everywhere & that's the only way we ever use this function 👍

Reply:
@Beamanator Yes that's the only way we ever use this function. We only use this to get the `submitsTo` of the expense report.

Reply:
Ok then I think we're good to go here 👍